### PR TITLE
Azure RT PR for Sprint 44

### PR DESCRIPTION
**Pull request by markcowl**

### CODE REVIEW

File: AzurePowershell.Test.targets [hunk 1]
```diff
@@ -21,13 +21,15 @@
...
+    <TestFilter>"!Functional&#x26;!Scenario&#x26;!AzureRTScenario&#x26;!Sequential&#x26;!PIRTest&#x26;!Preview&#x26;!ADDomain&#x26;!Network&#x26;!AzureRTUpload"</TestFilter>
     <KeyVaultTestDebug>.\src\ResourceManager\KeyVault\Commands.KeyVault.Test\bin\Debug\Microsoft.Azure.Commands.KeyVault.Test.dll</KeyVaultTestDebug>
-    <TestFilter>"!Functional&#x26;!Scenario&#x26;!AzureRTScenario&#x26;!Sequential&#x26;!PIRTest&#x26;!Preview&#x26;!ADDomain"</TestFilter>
     <ScenarioTestFilter>All</ScenarioTestFilter>
     <OneSDKCITFilter>"OneSDK&#x26;CIT"</OneSDKCITFilter>
-    <AzureRTAllTestFilter>"Functional|AzureRTScenario"</AzureRTAllTestFilter>
-    <AzureRTAllTestFilter2>"Functional|AzureRTScenario|Preview"</AzureRTAllTestFilter2>
-    <AzureRTAllTestFilter3>"Functional|AzureRTScenario|Sequential"</AzureRTAllTestFilter3>
+    <AzureRTAllTestFilter>"Functional|AzureRTScenario|Sequential|AzureRTUpload|Network"</AzureRTAllTestFilter>
+	<AzureRTParAllFilter>"Functional|AzureRTScenario"</AzureRTParAllFilter>
```
Comment (ogail) [inline AzurePowershell.Test.targets]: Fix alignment

File: src/Common/Commands.Common.Storage/Commands.Common.Storage.csproj [hunk 2]
```diff
@@ -70,12 +70,15 @@
     </Reference>
     <Reference Include="Microsoft.Data.Edm, Version=5.6.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
       <HintPath>..\..\packages\Microsoft.Data.Edm.5.6.0\lib\net40\Microsoft.Data.Edm.dll</HintPath>
+      <Private>True</Private>
```
Comment (ogail) [inline src/Common/Commands.Common.Storage/Commands.Common.Storage.csproj]: Remove these tags

File: src/ResourceManager/DataFactories/Commands.DataFactories/Commands.DataFactories.csproj [hunk 3]
```diff
@@ -69,6 +69,15 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\..\..\packages\Microsoft.Azure.Management.Resources.2.12.0-preview\lib\net40\Microsoft.Azure.ResourceManager.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.Data.Edm, Version=5.6.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```
Comment (ogail) [inline src/ResourceManager/DataFactories/Commands.DataFactories/Commands.DataFactories.csproj]: Why this project is changed it's not AzureRT's one

File: src/ResourceManager/DataFactories/Commands.DataFactories/Commands.DataFactories.csproj [hunk 4]
```diff
@@ -122,6 +131,7 @@
     </Reference>
     <Reference Include="System.Spatial, Version=5.6.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
       <HintPath>..\..\..\packages\System.Spatial.5.6.0\lib\net40\System.Spatial.dll</HintPath>
+      <Private>True</Private>
```
Comment (ogail) [inline src/ResourceManager/DataFactories/Commands.DataFactories/Commands.DataFactories.csproj]: Please make sure to remove all occurrences to `<Private>True</Private>`

File: src/ResourceManager/DataFactories/Commands.DataFactories/packages.config [hunk 5]
```diff
@@ -9,6 +9,9 @@
   <package id="Microsoft.Bcl" version="1.1.9" targetFramework="net45" />
   <package id="Microsoft.Bcl.Async" version="1.0.168" targetFramework="net45" />
   <package id="Microsoft.Bcl.Build" version="1.0.14" targetFramework="net45" />
+  <package id="Microsoft.Data.Edm" version="5.6.0" targetFramework="net45" />
```
Comment (ogail) [inline src/ResourceManager/DataFactories/Commands.DataFactories/packages.config]: Remove

File: src/ResourceManager/StreamAnalytics/Commands.StreamAnalytics/Commands.StreamAnalytics.csproj [hunk 6]
```diff
@@ -65,6 +65,15 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\..\..\packages\Microsoft.Azure.Management.StreamAnalytics.0.14.0-preview\lib\net40\Microsoft.Azure.Management.StreamAnalytics.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.Data.Edm, Version=5.6.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```
Comment (ogail) [inline src/ResourceManager/StreamAnalytics/Commands.StreamAnalytics/Commands.StreamAnalytics.csproj]: Same for all non AzureRT projects

File: src/ServiceManagement/Compute/Commands.ServiceManagement.Test/FunctionalTests/ServiceManagementTest.cs [hunk 7]
```diff
@@ -94,64 +94,64 @@ public static void AssemblyInit(TestContext context)
...
-                try
-                {
-                    vmPowershellCmdlets.RunPSScript("Get-AzureReservedIp | Remove-AzureReservedIp -Force");
-                }
-                catch
-                {
-                    Console.WriteLine("Error occurred during cleaning up reserved ips..");
-                }
-            }
+            //vmPowershellCmdlets = new ServiceManagementCmdletTestHelper();
```
Comment (ogail) [inline src/ServiceManagement/Compute/Commands.ServiceManagement.Test/FunctionalTests/ServiceManagementTest.cs]: Remove commented code if it's not used.

File: src/ServiceManagement/Compute/Commands.ServiceManagement.Test/UnitTests/Cmdlets/IaaS/Extensions/VirtualMachineExtensionImageFactoryTests.cs [hunk 8]
```diff
@@ -76,8 +76,8 @@ public void CleanupTest()
         {
         }
 
-        [TestMethod]
-        [TestCategory(Category.Functional)]
+        //[TestMethod]
```
Comment (markcowl) [inline src/ServiceManagement/Compute/Commands.ServiceManagement.Test/UnitTests/Cmdlets/IaaS/Extensions/VirtualMachineExtensionImageFactoryTests.cs]: do not comment the code, please remove the tests altogether

File: src/ServiceManagement/Compute/Commands.ServiceManagement.Test/FunctionalTests/StaticCATests.cs [hunk 9]
```diff
@@ -47,11 +47,11 @@ public static void Intialize(TestContext context)
         {
             imageName = vmPowershellCmdlets.GetAzureVMImageName(new[] { "Windows" }, false);
             var vnetConfig = vmPowershellCmdlets.GetAzureVNetConfig(null);
-            if (vnetConfig.Count > 0)
-            {
-                vmPowershellCmdlets.RunPSScript("Get-AzureService | Remove-AzureService -Force");
-                Utilities.RetryActionUntilSuccess(() => vmPowershellCmdlets.RemoveAzureVNetConfig(), "in use", 5, 30);
-            }
+            //if (vnetConfig.Count > 0)
```
Comment (markcowl) [inline src/ServiceManagement/Compute/Commands.ServiceManagement.Test/FunctionalTests/StaticCATests.cs]: Please do not comment out the cocde, please remove it

File: src/ServiceManagement/Compute/Commands.ServiceManagement.Test/FunctionalTests/LBandEndPointACLsTest.cs [hunk 10]
```diff
@@ -262,12 +262,13 @@ public void AzureEndpointLBDTest()
...
                     if (p == AzureEndPointConfigInfo.ParameterSet.NoLB)
                     {
-                        Console.WriteLine("-----Change the second endpoint.");
-                        ep2Info.EndpointLocalPort = ep2LocalPortChanged;
-                        ep2Info.EndpointPublicPort = ep2PublicPortChanged;
-                        ep2Info.LoadBalancerDistribution = LoadBalancerDistribution.None;
-                        vmPowershellCmdlets.SetEndPoint(defaultVm, serviceName, ep2Info); // Set-AzureEndpoint with Get-AzureVM and Update-AzureVm
-                        CheckEndpoint(defaultVm, serviceName, new[] { ep2Info });
+                        Console.WriteLine("Skipping None until the bug is fixed..");
+                        //Console.WriteLine("-----Change the second endpoint.");
```
Comment (markcowl) [inline src/ServiceManagement/Compute/Commands.ServiceManagement.Test/FunctionalTests/LBandEndPointACLsTest.cs]: Please do not comment out code, please remove it.

File: src/ServiceManagement/Compute/Commands.ServiceManagement.Test/FunctionalTests/FunctionalTest.cs [hunk 11]
```diff
@@ -472,11 +751,11 @@ public void AzureMultiNicTest()
 
                 // Create a VNet
                 var vnetConfig = vmPowershellCmdlets.GetAzureVNetConfig(null);
-                if (vnetConfig.Count > 0)
-                {
-                    vmPowershellCmdlets.RunPSScript("Get-AzureService | Remove-AzureService -Force");
-                    Utilities.RetryActionUntilSuccess(() => vmPowershellCmdlets.RemoveAzureVNetConfig(), "in use", 5, 30);
-                }
+                //if (vnetConfig.Count > 0)
```
Comment (markcowl) [inline src/ServiceManagement/Compute/Commands.ServiceManagement.Test/FunctionalTests/FunctionalTest.cs]: Please do not comment out code, please remove it.

File: src/ServiceManagement/Compute/Commands.ServiceManagement.Test/FunctionalTests/ExtensionTests/CustomScriptExtensionTests.cs [hunk 12]
```diff
@@ -195,7 +195,8 @@ private void GetCustomScriptExtensionVersion()
         {
             Utilities.PrintHeader("Listing the available VM extensions");
             var extensionsInfo = vmPowershellCmdlets.GetAzureVMAvailableExtension(ConstCustomScriptExtensionName, ConstCustomScriptExtensionPublisher, true);
-            customScriptExtension = extensionsInfo.OrderBy(c => c.Version).LastOrDefault();
+            //customScriptExtension = extensionsInfo.OrderBy(c => c.Version).LastOrDefault();
```
Comment (markcowl) [inline src/ServiceManagement/Compute/Commands.ServiceManagement.Test/FunctionalTests/ExtensionTests/CustomScriptExtensionTests.cs]: Please do not comment out code, please remove it

File: src/ServiceManagement/Compute/Commands.ServiceManagement.Test/FunctionalTests/InternaloadbalancerTests.cs [hunk 13]
```diff
@@ -42,11 +42,11 @@ public class InternalLoadBalancerTests : ServiceManagementTest
         public static void ClassInitialize(TestContext context)
         {
             var vnetConfig = vmPowershellCmdlets.GetAzureVNetConfig(null);
-            if (vnetConfig.Count > 0)
-            {
-                vmPowershellCmdlets.RunPSScript("Get-AzureService | Remove-AzureService -Force");
-                Utilities.RetryActionUntilSuccess(() => vmPowershellCmdlets.RemoveAzureVNetConfig(), "in use", 5, 30);
-            }
+            //if (vnetConfig.Count > 0)
```
Comment (markcowl) [inline src/ServiceManagement/Compute/Commands.ServiceManagement.Test/FunctionalTests/InternaloadbalancerTests.cs]: still commented code

File: src/ServiceManagement/Compute/Commands.ServiceManagement.Test/FunctionalTests/LocationBasedReservedIPTests.cs [hunk 14]
```diff
@@ -35,11 +35,11 @@ public class LocationBasedReservedIPTests: ServiceManagementTest
         public static void ClassInitialize(TestContext context)
         {
             var vnetConfig = vmPowershellCmdlets.GetAzureVNetConfig(null);
-            if (vnetConfig.Count > 0)
-            {
-                vmPowershellCmdlets.RunPSScript("Get-AzureService | Remove-AzureService -Force");
-                Utilities.RetryActionUntilSuccess(() => vmPowershellCmdlets.RemoveAzureVNetConfig(), "in use", 5, 30);
-            }
+            //if (vnetConfig.Count > 0)
```
Comment (markcowl) [inline src/ServiceManagement/Compute/Commands.ServiceManagement.Test/FunctionalTests/LocationBasedReservedIPTests.cs]: still commented code

**Pull request merged**